helpdesk_scope/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class HelpdeskTicket(models.Model):
+    _inherit='helpdesk.ticket'
+
+    scope = fields.Many2one('helpdesk.scope', string="Scope")
+    
```

Comment:
@osi-scampbell
 `scope_id = `
No need to import api.
No empty line at the end of the file.